kolibri/logger/csv.py

```diff
@@ -47,10 +47,10 @@ def get_content_title(self, obj):
             return ""
 
     def get_time_spent(self, obj):
-        return round(obj.time_spent, 3)
+        return "{:10.1f}".format(round(obj.time_spent, 1))
```

Comment:
Output would be `days:hours:minutes:seconds.tenthsofseconds`.

You may be able to use https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior for formatting purposes.